pkg/signing/signing.go

```diff
@@ -223,6 +367,70 @@ func SifAddSignature(sinfo *image.Sifinfo, signature []byte) error {
 	return nil
 }
 
+func sykeysFetchPubkey(fingerprint string, sykeysAddr string) (openpgp.EntityList, error) {
+	url := "http://" + sykeysAddr + "/pks/lookup?op=get&options=mr&search=0x" + fingerprint
+	log.Println("url :", url)
+	res, err := http.Get(url)
+	if err != nil {
+		log.Println(err)
+		return nil, err
+	}
+
+	el, err := openpgp.ReadArmoredKeyRing(res.Body)
+	res.Body.Close()
```

Comment:
Using `defer res.Body.Close()` recommended, close as possible to the `http.Get`.

Reply:
Will do !

Reply:
@tri-adam Quick question just for my own reference, why is using a `defer ...` statement generally preferred? Is this something that Go just says is best practice or is it to avoid mistakes?

Reply:
It just makes for easier maintenance, since the statement is grouped with the thing that it is related to, you don't have to hunt for it, and if there are early `return`s, you're not at risk of forgetting to `Close` something.